tests/rust-integration-tests/integration_test/src/tests/hostname/mod.rs

```diff
@@ -15,7 +15,7 @@ fn create_spec(hostname: &str) -> Spec {
         )
         .process(
             ProcessBuilder::default()
-                .args(vec!["runtimetest".to_string()])
+                .args(vec!["runtimetest".to_string(), "set_host_name".to_string()])
```

Comment:
Why this change (same question as adding clap in runtimetest)?

Reply:
Same above comment.

Reply:
Makes sense, just see reply to the above.